shared/chat/inbox/row/chat-filter-row.js

```diff
@@ -120,16 +110,48 @@ class _ChatFilterRow extends Component<Props, State> {
   }
 }
 
+const styleContainer = {
+  ...globalStyles.flexBoxRow,
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  minHeight: 48,
+  paddingLeft: globalMargins.small,
+  paddingRight: globalMargins.small,
+  position: 'relative',
+}
+
+const styleContainerMobile = {
```

Comment:
this looks like it doesn't change `styleContainer`. Did you mean it to be different? If not, remove?

Reply:
Ah yes! Removed.